docs/notes/2.22.x.md

```diff
@@ -68,6 +68,10 @@ Setting [the `orphan_files_behaviour = "ignore"` option](https://www.pantsbuild.
 
 ### Plugin API changes
 
+## Bugfixes
+
+- Remote cache: `FindMissingBlobsRequest` will now make multiple request if the number of files is large. (#20708)
```

Comment:
Similar to https://github.com/pantsbuild/pants/pull/20492#discussion_r1599202471:

- we've historically put this sort of change into a dedicated "remote caching/execution" section: https://github.com/pantsbuild/pants/blob/main/docs/notes/2.20.x.md#remote-cachingexecution
- we might want to make the PR reference a manual link